src/daemon/BUILD.gn

```diff
@@ -62,6 +62,9 @@ source_set("configuration") {
 }
 
 protobuf("config_proto") {
+  deps = [
```

Comment:
This can be done as follows:

```
malets@ub:~/dev/dist-clang$ rm -rf out/Test.gn/
malets@ub:~/dev/dist-clang$ ./build/configure
Done. Wrote 102 targets from 23 files in 71ms
Done. Wrote 102 targets from 23 files in 66ms
Done. Wrote 106 targets from 25 files in 67ms
malets@ub:~/dev/dist-clang$ ninja -C out/Test.gn obj/out/Test.gn/gen/src/daemon/config_proto.configuration.pb.o

...

[269/269] CXX obj/out/Test.gn/gen/src/daemon/config_proto.configuration.pb.o
FAILED: clang++ -MMD -MF obj/out/Test.gn/gen/src/daemon/config_proto.configuration.pb.o.d -D_DEBUG -DOS_LINUX -I../../src -Igen/src -I../../src/third_party/protobuf/exported/src -I../../src/third_party/protobuf/linux -I../../src/third_party/libcxx/exported/include -I../../src/third_party/libcxxabi/exported/include -fsanitize=address -fno-omit-frame-pointer -fPIC -pipe -pthread -Wall -Wsign-compare -Wno-unknown-warning-option -Wno-error=assume -Wno-error=unused-command-line-argument -Werror -g -O0 -Wno-bitfield-width -std=c++1z -nostdinc++ -c gen/src/daemon/configuration.pb.cc -o obj/out/Test.gn/gen/src/daemon/config_proto.configuration.pb.o
In file included from gen/src/daemon/configuration.pb.cc:5:
gen/src/daemon/configuration.pb.h:27:10: fatal error: 'base/base.pb.h' file not found
#include "base/base.pb.h"
         ^
1 error generated.
ninja: build stopped: subcommand failed.
```